src/site/_includes/components/PathCard.js

```diff
@@ -29,6 +29,16 @@ function getPostCount(learningPath) {
   // in path.njk. Ideally we should do this in the learningPath .11ty.js files
   // but eleventy hasn't parsed all of the collections when those files get
   // initialized so we can't look up posts by slug.
+
+  // Merge subTopic pathItems
+  learningPath.topics = (learningPath.topics).map((topic) => {
+    const subPathItems = (topic.subTopics || []).reduce((accumulator, subTopic) => {
+      return ([...accumulator, ...(subTopic.pathItems)]);
+    }, []);
+    topic.pathItems = [...subPathItems, ...(topic.pathItems || [])];
```

Comment:
My understanding is this will put subpath items before path items. I wonder if we want to reverse it so pathItems happen first. I think that would allow folks to include intro posts, similar to this:

## Topic: Fast
- An introduction to performance

### Subtopic: Measuring speed
- Something about measuring

### Subtopic: Optimizing speed
- Something about optimizing